### PR TITLE
Admin movie details

### PR DESCRIPTION
**Pull request by mazonk**
Pull request for admin section - movie management. 
Currently you are able to create, edit and delete a movie. However the imageUploader is not included and also the movie does not have director, actors and genres co-queried in the repository. That will be included in the upcoming PR.

### CODE REVIEW

**Review by TLNRB — APPROVED** [review]
Looks lovely, nice work.

=======
<<<<<<< Updated upstream
=======

There are comments/thingies like this, maybe it is not meant to be there? (e.x. line 421)

File: routes.php [hunk 1]
```diff
@@ -418,3 +418,141 @@
         echo json_encode(['success' => false, 'errorMessage' => 'Invalid action.']);
     }
 });
+
+//Add movie post route
+post($baseRoute.'movies/add', function() {
+    require_once 'src/controller/MovieController.php';
+    $movieController = new MovieController();
+    parse_str(file_get_contents("php://input"), $_POST); // Parse the PUT request
```
Comment (4ndris1114) [inline routes.php:426]: think this is unnecessary on a POST request (to my best knowledge it's only necessary for PUT requests)

File: src/view/components/admin-sections/movies/movieCRUD.php [hunk 2]
```diff
@@ -0,0 +1,597 @@
...
+        addMovieIdInput.value = id;
+        addMovieModal.classList.remove('hidden');
+    };
+
+    // Add movie form submission
+    addMovieForm.addEventListener('submit', function (event) {
+        event.preventDefault();
+        const xhr = new XMLHttpRequest();
+        const baseRoute = '<?php echo $_SESSION['baseRoute']; ?>';
+        console.log('<?php echo $_SESSION['baseRoute']; ?>');
```
Comment (4ndris1114) [inline src/view/components/admin-sections/movies/movieCRUD.php:257]: would be nice to get rid of console.logs

**Review by 4ndris1114 — APPROVED** [review]
super nice job done <3

File: routes.php [hunk 3]
```diff
@@ -418,8 +418,61 @@
         echo json_encode(['success' => false, 'errorMessage' => 'Invalid action.']);
     }
 });
+<<<<<<< Updated upstream
```
Comment (4ndris1114) [inline routes.php]: github conflict?

**Pull request merged**